spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/ConfigMapConfigProperties.java

```diff
@@ -85,7 +85,7 @@ public void setSources(List<Source> sources) {
 			return Collections.singletonList(new NormalizedSource(name, namespace, ""));
 		}
 
-		return sources.stream().map(s -> s.normalize(name, namespace, useNameAsPrefix, s.getExplicitPrefix()))
```

Comment:
I've realized this when writing code for a different defect. since I we are calling `normalize` of the `Source` already (`s`), passing `s.getExplicitPrefix` kind of makes no sense. I can do that via `this.explicitPrefix` already. 

minor thing, but you know, makes total sense. I guess.